cmd/hierarchy-transformer/output/businessInvestment/hierarchy.cypher

```diff
@@ -0,0 +1,175 @@
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "M" and child.code = "M71" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "M" and child.code = "M72" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "M" and child.code = "M73" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "M" and child.code = "M73T75" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "M" and child.code = "M74_75" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "_T" and child.code = "M_N" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "M_N" and child.code = "N" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "N" and child.code = "N77" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "N" and child.code = "N78" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "N" and child.code = "N79" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "N" and child.code = "N80T82" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "_T" and child.code = "O" CREATE (child)-[:hasParent]->(parent);
+MATCH (parent:`_generic_hierarchy_node_business-investment-activity`), (child:`_generic_hierarchy_node_business-investment-activity`) where parent.code = "O" and child.code = "OTQ" CREATE (child)-[:hasParent]->(parent);
```

Comment:
OTQ is a child of O? But not a child of P? Shouldnt O, P and Q each be a child of OTQ?